physi.js

```diff
@@ -31,42 +30,44 @@ window.Physijs = (function() {
 
 	Physijs.scripts = {};
 
-	Eventable = function() {
-		this._eventListeners = {};
-	};
-	Eventable.prototype.addEventListener = function( event_name, callback ) {
-		if ( !this._eventListeners.hasOwnProperty( event_name ) ) {
-			this._eventListeners[event_name] = [];
+	const Eventable = (superclass) => class extends superclass {
```

Comment:
Following this mix-in style: https://stackoverflow.com/questions/29879267/es6-class-multiple-inheritance

So that we can use this syntax: `class Scene extends Eventable(THREE.Scene)`